### DISCUSSION TITLE
Display mentioned labels

### DISCUSSION THREAD

**Discussion by VoltrexKeyva** [community/community#44083] [Mobile]
On GitHub you can copy the link to a label of a repository in the form of `https://github.com/<owner>/<repository>/labels/<label name>` (e.g. https://github.com/nodejs/node/labels/semver-major) and use them in comments and such to mention the label.

GitHub displays the labels that are mentioned on the website:
![Screenshot_20230112-130240_Chrome.jpg](https://user-images.githubusercontent.com/62040526/212032040-0b02edd3-fa5d-4005-8130-e9796ea742f1.jpg)

However it doesn't display it on GitHub Mobile:
![Screenshot_20230112-130326_GitHub.jpg](https://user-images.githubusercontent.com/62040526/212032083-34204c5f-677e-457e-bdfd-2a55285e0887.jpg)

It rather shows a hyperlink to the label's link along with it's description, which adds unwanted text to the comment and such which can be confusing to the reader.

Link to the comment in the screenshots if anyone wants to test: https://github.com/nodejs/node/pull/46174#issuecomment-1379960508

**Comment by alcere** [comment]
Hi @VoltrexKeyva thanks for the suggestion! I am passing the feedback to the team and they will evaluate the idea!

**Reply by VoltrexKeyva** [reply]
Any updates on this?

**Reply by Dragon1573** [reply]
I'm using _GitHub Mobile v1.165.0-beta (10194)_ at the moment. This issue is still there. 😕

- https://github.com/microsoft/winget-pkgs/issues/157583#issuecomment-2174895379

![image](https://github.com/community/community/assets/49941141/611d29cb-aa69-42d1-9945-e8f16456e7ed)
![Screenshot_20240618_224553_com github android](https://github.com/community/community/assets/49941141/36b3d686-8ef3-43be-88e0-b78a6f9d0e7b)

Is there any **Projects** (no matter classic or not), **Milestones** or **Pull Requests** related to this issue so we can visualize the progress?

-----

Edit01: Everything keep the same even after upgraded to _v1.166.1 (10195)_
